web/js/modules/layers/actions.js

```diff
@@ -391,3 +393,24 @@ export function changeGranuleSatelliteInstrumentGroup(id, granulePlatform) {
     });
   };
 }
+
+export function updateLayerCollection(value) {
+  return (dispatch, getState) => {
+    const { layers } = getState();
+    const { id } = value;
+    const collections = layers.collections[id];
+    if (!collections) {
+      dispatch({
+        type: UPDATE_LAYER_COLLECTION,
+        id,
+      });
+    }
+  };
+}
+
+export function updateLayerDateCollection(value) {
```

Comment:
Same as above. Can we provide a function definition?

Reply:
Renamed this parameter to `layerInfo`